src/Klein/Request.php

```diff
@@ -290,7 +290,7 @@ public function params($mask = null, $fill_with_nulls = true)
      *
      * @param string $key       The name of the parameter to return
      * @param mixed $default    The default value of the parameter if it contains no value
-     * @return string
+     * @return string|array|null
```

Comment:
According to https://www.phpdoc.org/docs/latest/guides/types.html type may be mixed, if "the author of the documentation is unable to predict which type it will be". 

At the same time variable may have "multiple types combined", when "an element may accept or return a value that can be any of a limited set of Types".

If return type may be not in (string, array, null), it may be or appended, or declared as mixed :)